ArduCopter/mode_guided.cpp

```diff
@@ -84,8 +84,27 @@ void ModeGuided::run()
     case SubMode::Angle:
         angle_control_run();
         break;
+
+    case SubMode::Circle:
+        switch (loiter_turns.get_state()) {
+        case _LoiterTurns::State::CIRCLING: {
+            loiter_turns.circle_run();
+            break;
+        }
+        case _LoiterTurns::State::MOVING_TO_EDGE:
+            // use waypoint navigation controller in this state
+            wp_control_run();
+            break;
+        default:
+            break;
```

Comment:
```suggestion
```

if there are extra states then they should be put in here - even if they just `break;`